GitExtensions.Core/GitExtensions.Core.csproj

```diff
@@ -0,0 +1,5 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why do we need this project?
Any new projects within the solution must be added to all the installer scripts. This is an overhead which needs a good justification.

Reply:
This project is for basic interfaces and application mechanisms.
This does not apply directly to plugins.
But we can place this project in https://github.com/gitextensions/gitextensions.extensibility too.
It seems to me best not to mix this directly with the plugin interfaces.

Reply:
> This project is for basic interfaces and application mechanisms.

If this is app-specific, let's hold off creating new projects, and keep it in on of the existing assemblies. Once we switch to .NET we can create new projects, I expect our deployment story will change dramatically with the migration.

Reply:
These projects are migration only and will be removed from the main repository.